.github/workflows/hacs.yaml

```
@@ -0,0 +1,17 @@
+name: HACS Action
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  hacs:
+    name: HACS Action
+    runs-on: "ubuntu-latest"
```

As a safety precaution, I would specify [required permissions](https://docs.github.com/en/actions/how-tos/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token):
```suggestion
    name: HACS Action
    runs-on: "ubuntu-latest"
    
    permissions:
       contents: read
```